ficlatte/settings-example.py

```diff
@@ -1,106 +0,0 @@
-"""
-Django settings for ficlatte project.
```

Comment:
All I did was rename it to settings. Probably should have just copied it and renamed the copy, now that I think about it.